test/examples/test_bart.py

```diff
@@ -31,11 +32,10 @@ def bounded_exp(x, bound):
 call_count = 0
 
 
-@funsor.function(
-    Reals[2 * num_origins * num_destins],
-    (Reals[num_origins, num_destins, 2], Reals[num_origins, num_destins]),
-)
-def unpack_gate_rate(gate_rate):
+@funsor.function
+def unpack_gate_rate(
+    gate_rate: Reals[2 * num_origins * num_destins],
+) -> typing.Tuple[Reals[num_origins, num_destins, 2], Reals[num_origins, num_destins]]:
```

Comment:
Does this syntax already work, or does this `Tuple` need to be a `funsor.domains.Product`?

Reply:
The tests pass 🤷